source/systems/BannerNotifications/BannerNotificationsComponent.js

```diff
@@ -8,7 +8,7 @@ const mapStateToProps = state => ({
     messages: state.BannerNotificationsStore.getNotifications().toJS(),
 });
 
-const mapDispatchToProps = dispatch => ({
+const mapDispatchToProps = /* istanbul ignore next */ dispatch => ({
```

Comment:
Why did you have to ignore this test?

Reply:
Apparently the disable that's already inside the function wasn't enough - the `onClose` function was reported as untested. I updated jest and some other libs so that might be the issue.